test/utils.test.jest.js

```diff
@@ -0,0 +1,199 @@
+'use strict';
+
+/* eslint-disable unicorn/new-for-builtins, no-new-wrappers, prefer-regex-literals, jest/prefer-expect-assertions */
```

Comment:
In the same time some test cases will look as 
```
expect.hasAssertions();
expect....
```
In this case we can ignore that rule. Or copy-paste `expect.hasAssertions();` in all tests. All the variants are good

Reply:
Not sure I follow what you mean?

Reply:
@XhmikosR I am talking about this rule https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/prefer-expect-assertions.md. I commented about it here https://github.com/svg-sprite/svg-sprite/pull/572#discussion_r825446342
So there is two options
1. `expect.hasAssertions` in every test case
2. disabling this rule in some test cases like in this one

Reply:
Hmm, both work for me but let's try to keep it simple for future contributors :)